test/rise-page-ready.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>Page - Ready</title>
+
+  <script src="../../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script>
+    // This needs to run before any timers are set.
+    var clock = sinon.useFakeTimers();
```

Comment:
@stulees Maybe? The symptoms were the same in that the timer was not firing until I moved this up here. We should try this out next time we're in those tests.